### ISSUE TITLE
Data.format.parse does not support 'utc:"%m%d%Y"

### ISSUE THREAD

**Issue by YuhanLu** [vega/vega#818]
From reading the [doc](https://vega.github.io/vega/docs/data/), I think vega support `parse: {field: 'utc:"%m%d%Y"}` or `parse: {field: 'date:"%m%d%Y"}`. I have tried a couple of examples and they did work but when I tried to add an example in vl, the validator failed at the line `"parse": {"date": "utc:%d %b %Y %H:%M:%S"}`. Could you please help to fix it?

**Comment by domoritz** [comment]
Can you try `"parse": {"date": "utc:'%d %b %Y %H:%M:%S'"}`?

**Comment by YuhanLu** [comment]
Yah I did try those with/without `'`. I think the problem is about the json schema.

**Comment by domoritz** [comment]
Is that a problem in vega or vega-lite? Can you post the schema validation error? Also, can you post an example so I can reproduce this issue?

**Comment by YuhanLu** [comment]
It's in vega and here is the spec:

```
{
  "$schema": "http://vega.github.io/schema/vega/v3.0.json",
  "description": "Google's stock price over time.",
  "autosize": "pad",
  "padding": 5,
  "data": [
    {
      "name": "source_0",
      "values": [
        {"date": "10 Oct 2011 22:48:00"},
        {"date": "11 Oct 2022 23:00:00"}
      ],
      "format": {
        "type": "json",
        "parse": {"date": "utc:%d %b %Y %H:%M:%S"}
      },
      "transform": [
        {
          "type": "filter",
          "expr": "datum[\"date\"] !== null && !isNaN(datum[\"date\"])"
        },
        {
          "type": "formula",
          "as": "hours_date",
          "expr": "datetime(0, 0, 1, hours(datum[\"date\"]), 0, 0, 0)"
        }
      ]
    }
  ],
  "signals": [
    {"name": "width","update": "21"},
    {
      "name": "height",
      "update": "bandspace(domain('y').length, 1, 0.5) * 21"
    }
  ],
  "marks": [
    {
      "name": "nested_main_group",
      "type": "group",
      "encode": {
        "update": {
          "width": {"signal": "width"},
          "height": {"signal": "height"},
          "fill": {"value": "transparent"}
        }
      },
      "marks": [
        {
          "name": "marks",
          "type": "symbol",
          "role": "point",
          "from": {"data": "source_0"},
          "encode": {
            "update": {
              "x": {"value": 10.5},
              "y": {"scale": "y","field": "hours_date"},
              "stroke": {"value": "#4c78a8"},
              "fill": {"value": "transparent"},
              "opacity": {"value": 0.7}
            }
          }
        }
      ]
    }
  ],
  "scales": [
    {
      "name": "y",
      "type": "point",
      "domain": {
        "data": "source_0",
        "field": "hours_date",
        "sort": true
      },
      "range": {"step": 21},
      "round": true,
      "padding": 0.5
    }
  ],
  "axes": [
    {
      "scale": "y",
      "orient": "left",
      "title": "time",
      "zindex": 1,
      "encode": {
        "labels": {
          "update": {
            "text": {"signal": "timeFormat(datum.value, '%H')"}
          }
        }
      }
    }
  ]
}
```

Thanks!

**Comment by jheer** [comment]
This has nothing to do with `utc` per se, but is a bug in the schema definition for parsing rules. I'll fix that over in the vega-parser repo. Thanks!

**Comment by jheer** [comment]
Fixed in vega/vega-parser@cfc2c88cb618f9fc7f5619fa5e1120729c1fc9f7 and released with vega-parser v1.0.0-beta.54.

**Comment by YuhanLu** [comment]
Thank you!
